daprdocs/content/en/reference/components-reference/supported-bindings/wasm.md

```diff
@@ -80,18 +83,39 @@ pass metadata properties with each request:
 
 - `args` any CLI arguments, comma-separated. This excludes the program name.
 
-For example, if the binding `url` was a Ruby interpreter, such as from
-[webassembly-language-runtimes](https://github.com/vmware-labs/webassembly-language-runtimes/releases/tag/ruby%2F3.2.0%2B20230215-1349da9),
-the following request would respond back with "Hello, salaboy":
+For example, consider binding the `url` to a Ruby interpreter, such as from
+[webassembly-language-runtimes](https://github.com/vmware-labs/webassembly-language-runtimes/releases/tag/ruby%2F3.2.0%2B20230215-1349da9):
 
-```json
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: wasm
+spec:
+  type: bindings.wasm
+  version: v1
+  metadata:
+  - name: url
+    value: "https://github.com/vmware-labs/webassembly-language-runtimes/releases/download/ruby%2F3.2.0%2B20230215-1349da9/ruby-3.2.0-slim.wasm"
+```
+
+and assuming that you started your Wasm Binding at port 3500:
+
+```
+$ dapr run --app-id wasm --dapr-http-port 3500 --resources-path components
+```
+
+then, the following request will respond back with "Hello, salaboy":
```

Comment:
```suggestion
The following request responds with "Hello, salaboy":
```

Reply:
One more thing, since you have changed the args, the respond should be "Hello salaboy".

Reply:
it's more precisely `Hello "salaboy"`